src/main/content/_assets/js/builds.js

```diff
@@ -1591,3 +1591,12 @@ $(window).on('load', function () {
         }
     });
 });
+
+$(window).on('load', function (e) {   
+    if(window.location.hash) {
```

Comment:
Please move lines 1596-1601 to the above `$(window).on(load, ...)` method, below line 1592

Reply:
@navaneethsnair1 please apply these changes when you get a chance.